dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracle.scala

```diff
@@ -119,6 +119,30 @@ case class DLCOracle(private val extPrivateKey: ExtPrivateKeyHardened)(implicit
     }
   }
 
+  def createNewRangedEvent(
+      eventName: String,
+      maturationTime: Instant,
+      start: Int,
+      stop: Int,
+      step: Int = 1): Future[EventDb] = {
+    if (step == 0) {
```

Comment:
`step <= 0`

Reply:
negative step is okay ie 

start = 100
stop = 0
step = -1